File-Formats/PDF/Working-with-OCR/Working-with-OCR.md

```diff
@@ -254,6 +254,9 @@ Download the language packages from the following link.
 
 ## Get Started with OCR 
 
+To quickly get started with Extract Text from Scanned PDF in .NET using the OCR Library. Please, check this video:
```

Comment:
To quickly get started with extracting text from scanned PDF documents in .NET using the Syncfusion OCR processor Library, refer to this video tutorial: